src/EnrollCallAndSmsController.js

```diff
@@ -101,7 +101,7 @@ function (Okta, FormController, Footer, PhoneTextBox, TextBox, CountryUtil, Form
         self.trigger('errors:clear');
 
         if(!phoneNumber.length) {
-          self.trigger('invalid', self, {'phoneNumber': 'model.validation.field.blank'});
+          self.trigger('invalid', self, {'phoneNumber': 'model.validation.this.field.blank'});
```

Comment:
if there is anyone providing a custom message for `model.validation.field.blank`, it won't work anymore. So I think we need to release note that we change the `key` name, so when someone updates to the latest sign in widget, they are aware (@rchild-okta is this right? Or de we need some more communication about it)

Reply:
That's correct - if we're changing the property names, it would be a breaking change (exactly as you're describing).

+ @haishengwu-okta @robertdamphousse-okta

Reply:
good catch!!
Assume you're talking about developer override key via https://github.com/okta/okta-signin-widget#language-and-text.
we probably have to do so.

cc @nbarbettini

Reply:
yes, that is the one I am talking about.

Reply:
@haishengwu-okta @rchild-okta @mauriciocastillosilva-okta Imho, if that is the case, it is more worth to do an i18n breaking change keeping the name of the key and changing the value (assuming the value has to change for every occurrence) rather than doing a breaking change to the widget changing the key, so we are not affecting any external developers. CC: @robertdamphousse-okta @nbarbettini